apps/web/src/routes/_content/resources/conferences/$conferenceName-$conferenceId.tsx

```diff
@@ -161,64 +160,72 @@ function Conference() {
-            <div className="lg:order-2 w-full max-w-full">
-              <img
-                src={assetUrl(conference.path, 'thumbnail.webp')}
-                alt={conference.name}
-                className="w-full object-cover aspect-[915/388] rounded-2xl"
-              />
-            </div>
-            <div className="lg:max-w-[560px] lg:order-1 text-white w-full">
-              <h2 className="text-white mobile-h2 sm:text-[40px] sm:leading-tight sm:tracking-[0.25px]">
-                {conference.name}
-              </h2>
-              <span className="text-newGray-4 mobile-body2 sm:desktop-h8">
-                {conference.location}, {conference.year}
-              </span>
-              {(conference.twitterUrl || conference.websiteUrl) && (
-                <>
-                  <div className="flex flex-wrap items-center gap-4 mt-2">
-                    {conference.twitterUrl && (
-                      <a
-                        href={conference.twitterUrl}
-                        target="_blank"
-                        rel="noopener noreferrer"
-                      >
-                        <BsTwitterX className="size-4 md:size-6" />
-                      </a>
-                    )}
-                    {conference.websiteUrl && (
-                      <a
-                        href={conference.websiteUrl}
-                        target="_blank"
-                        rel="noopener noreferrer"
-                      >
-                        <BsLink className="size-4 md:size-6" />
-                      </a>
-                    )}
-                  </div>
-                </>
-              )}
-              <div className="flex flex-wrap items-center gap-4 mt-4">
-                {conference.tags.map((tag) => (
-                  <Tag key={tag} className="capitalize">
-                    {tag}
-                  </Tag>
-                ))}
+          <Card
+            className="md:mx-auto w-full !rounded-[10px] !md:rounded-[20px]"
+            withPadding={false}
+            paddingClass="p-5 md:p-[30px]"
+            color="orange"
+          >
+            <div className="flex flex-col lg:flex-row justify-center items-center w-full gap-5 lg:gap-10">
+              <div className="lg:order-2 w-full max-w-full">
+                <img
+                  src={assetUrl(conference.path, 'thumbnail.webp')}
+                  alt={conference.name}
+                  className="w-full object-cover aspect-[915/388] rounded-2xl"
+                />
+              </div>
+              <div className="lg:max-w-[560px] lg:order-1 text-white w-full">
+                <h2 className="text-white body-medium-16px md:display-medium-40px">
+                  {conference.name}
+                </h2>
+                <span className="text-newGray-4 label-medium-16px sm:desktop-h8">
+                  {conference.location}, {conference.year}
```

Comment:
Place and date shouldn't be comma separated (it's a dot on figma)
![image](https://github.com/user-attachments/assets/a2aa914e-1bc3-4956-83ac-b7a049dbdd78)